src/index.js

```diff
@@ -488,6 +501,15 @@ export default class AuthnWidget {
   registerMfaEventHandler() {
     Array.from(document.querySelectorAll('[data-mfa-selection]'))
       .forEach(element => element.addEventListener('click', this.handleMfaDeviceSelection));
+
+    Array.from(document.querySelectorAll('[data-mfa-selection-kebab-menu-container]'))
+      .forEach(element => element.addEventListener('click', this.showDeviceManagementPopup));
+
+    Array.from(document.querySelectorAll('div'))
+      .forEach(element => element.addEventListener('click', this.hideDeviceManagementPopup));
```

Comment:
Should we just bind this once globally? Something like document.addEventListener

Reply:
Updated code for this.